src/teb_local_planner_ros.cpp

```diff
@@ -1081,10 +1081,10 @@ RobotFootprintModelPtr TebLocalPlannerROS::getRobotFootprintFromParamServer(cons
 
     // check parameters
     XmlRpc::XmlRpcValue footprint_xmlrpc;
-    if (!nh.getParam("footprint_model/vertices", footprint_xmlrpc) )
+    if (!nh.getParam("footprint_model/vertices", footprint_xmlrpc) && (!nh.getParam("vertices/footprint", footprint_xmlrpc) ))
```

Comment:
does this always overwrite the first check?

Reply:
it does. But we should always have either of them present. I can do more if else statement and make it all separate. I didn't bother doing it because this change was just to keep it backwards compatible with upstream.